src/styles/blocks/card/blockscard/body.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
It's not a good practice to set styles directly to the `html` tag. It's better to set up global styles in a separate file, for example, `main.scss` or `global.scss`. Also, this file is supposed to contain styles related to the `body` class of the `card` block as per BEM methodology, so these styles are not related to this file.